packages/core/parcel/src/cli.js

```diff
@@ -418,6 +430,35 @@ async function normalizeOptions(command): Promise<InitialParcelOptions> {
     command.detailedReport = '10';
   }
 
+  let additionalReporters = [
+    '@parcel/reporter-cli',
+    '@parcel/reporter-dev-server',
+  ].map(packageName => ({packageName, resolveFrom: __dirname}));
+
+  reporterOuter: for (let name of (command.reporter: Array<string>)) {
+    let potentialNames = [
+      name,
+      `@parcel/reporter-${name}`,
+      `parcel-reporter-${name}`,
+    ];
```

Comment:
Hmmmm what do we think about this? We don't support these shorthands anywhere else...

Reply:
(I did this because `parcel build src/index.html --reporter @parcel/reporter-bundle-buddy` can lead to either very long commands and it's just faster to type just the last part for a one-off build. But I don't feel strongly about it)

Reply:
I'm ok with it just pointing out the difference. Maybe post in slack and get other opinions?